solana/program_v2/programs/gateway_v2/src/util.rs

```diff
@@ -47,15 +54,47 @@ pub fn calculate_network_and_gatekeeper_fee(fee: u64, split: u16) -> (u64, u64)
     (network_fee as u64, gatekeeper_fee as u64)
 }
 
+pub fn create_and_invoke_transfer<'a>(
```

Comment:
Is there a better way to do this? Not sure if I am using the lifetime trait correctly 🤔